### ISSUE TITLE
how to test the Cirfar10 

### ISSUE THREAD

**Issue by xujinchang** [chenyuntc/pytorch-GAN#1]
Good work! I have also learned GAN based on pytorch. The Cifar10 datasets has ten classes. Before using GAN, i trained Cifar10 using the following code:
for i, data in enumerate(trainloader, 0):
		inputs, labels = data
		inputs, labels = Variable(inputs.cuda()), Variable(labels.cuda())
		optimizer.zero_grad()
		outputs = net(inputs)
		loss = criterion(outputs, labels)
the labels ranges from 0 to 9. and test the network, we can using these code:
for i, data in enumerate(testloader, 0):
    images, labels = data
    images = Variable(images.cuda())
    labels = labels.cuda()
    outputs = net(images)
    _, predicted = torch.max(outputs.data, 1)
    total += labels.size(0)
    predicted = predicted.cuda()
    correct += (predicted == labels).sum()
However, With using the GAN network trained for Cifar10, the loss = criterion(outputs, labels) has changed to criterion(outputs, real_data) or criterion(outputs, fake_data) , the ten classifications has become two classifications. So how can i correctly predicted an image belonging to which class using the Gan network?
Thanks a lot.

**Comment by chenyuntc** [comment]
You may want to try [InfoGAN](https://arxiv.org/abs/1606.03657) .
